content/adopting-an-ethical-license.md

```diff
@@ -0,0 +1,41 @@
+
+## A Potentially Useful Framing
+
+An analogy that may (or may not!) be useful as you move forward is to think about intellectual property rights in the same manner of other property rights. Let's pretend a project is a "neighborhood." Each "house" has a "lock" on it that the "homeowner" installed when it was "built". This "lock" is opened with the right "key." If, as the "neighborhood maintainer", we want to change the lock, we _must_ check with the homeowner; unless they have given prior consent.
+
+The "homeowner" is the contributor. The "house" is a substantial contribution. The "lock" is the license, and the constraints the license imposes is the "key" to granting you the right to use the home.
+
+## Authorship Rights
+
+Authorship rights are often held by the original author of the contributed source code. Exceptions exist, such as when contributors explicitly assign authorship to the person(s) or organization(s) that maintain the project through a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement).
+
+Out of respect for these past authors, and to ensure adoption of ethical licenses proceed in a way that maintains the trust and integrity of the community; we believe maintainers _should_ acquire positive consent from all authors whose code remains within the project. Further, we recommend projects continue to conform to the constraints of their existing license until clear, positive, informed affirmative consent from past contributors is reached.
+
+## Existing License Constraints
+
+Most open source and free software licenses are widely permissive regarding how others may use the underlying code. However, that permission may have additional constraints applied. Many require that the license the author submitted the contribution under is distributed along with any copies of the software. Others require that any projects they interact with also release their source code in a license that conforms to particular rigors. In all cases, we recommend maintainers read and understand their existing licenses constraints.
+
+While we may not have a guide in place for the particular license your project is under, you may be able to determine a reasonable and socially constructive course of action from reviewing the strategies recommended for software written under other licenses.
+
+### The MIT License
+
+The MIT License is one of the most widely used open source licenses, and also one of the most permissive. This makes it one of the easiest to relicense, but it still may require a significant amount of work.
+
+The core constraint of the MIT license is that the body of the license must be included in any copy or derivative works. This constraint persists so long as any code that was contributed under the license remains in the project _or_ every author with code in the project agrees to relicense their contributions under the new license.
```

Comment:
I think this needs a slight wording change, since it sounds like you can move forward without a 100% response.  For example, it sounds like code from author "A" can't be relicensed until code by author "B" is acknowledged.  However, if everyone except Author "B" accepts, you can still move forward, provided the code from Author "B" is removed.

Reply:
My understanding is that you cannot _relicense_ without 100% response; so people who wish to adopt a non-MIT license without getting unanimous consent must "sublicense;" which basically indicates that the license for _some_ of the code is MIT and _some_ of the code is a different license. See https://github.com/vcr/vcr/pull/805 for an example of sublicensing.

That said, I agree that that is the least clear part of the guide at this time :).